docs/references/cli.md

```diff
@@ -128,6 +128,57 @@ optional arguments:
                      warning, error. Default: info
 ```
 
+### `upgrade`
+
+Upgrades an Opstrace cluster.
```

Comment:
I followed the same pattern as the commands above. @triclambert wdyt?

Reply:
`upgrade`  raises some important questions that the reference text should address.

Going back to my points in the first discussion/design doc, there are at least two imminent questions to be answered with respect to the _input_ of the operation:

- how is the target version defined? (the version of the CLI with which this command is executed defines the target version -- that might seem obvious, but is worth stating explicitly)
- why does the cluster config file need to be provided "again", compared to the initial creation? What if the contents differ compared to the one provided during the initial `create`?

Also, it would be great to summarize in two or three sentences how the upgrade is performed, and maybe how much downtime is to be expected.

Reply:
Thank you for the review. Added a few notes below.